cmd/generate.go

```diff
@@ -171,27 +177,33 @@ func runWithStdout(
 		return fmt.Errorf("failed to check if model has been updated: %w", err)
 	}
 	if updated {
-		targetPostgres, targetConn := setupDatabase(ctx, "target", 5432)
+		targetPostgres, targetConn, err := setupDatabase(ctx, "target", 5432)
 		defer func() {
 			_ = targetConn.Close(ctx)
 			_ = targetPostgres.Stop()
```

Comment:
Nitpick: both `targetConn` and `targetPostgres` can be `nil` here as there might have been an error